apps/wallet-mock/src/test/transaction.test.ts

```diff
@@ -260,3 +256,63 @@ test.describe('Transaction review modal -- Reject', () => {
     await page.close()
   })
 })
+
+test.describe('Transaction history tests', () => {
+  let page: Page
+  test.beforeEach(async ({ browser }) => {
+    page = await browser.newPage()
+    await mock(page)
+    await page.goto('/')
+    await beginInteractionSession(page, 'TRANSACTION_REVIEW')
+    await sendBackendInteraction(
+      page,
+      'REQUEST_TRANSACTION_REVIEW_FOR_SENDING',
+      data
+    )
+  })
+
+  test.afterEach(async () => {
+    await page.close()
+  })
+
+  test('should list rejected transactions', async () => {
+    //This test will fail until https://github.com/vegaprotocol/vegawallet-ui/issues/162 is fixed
+    // 0001-WALL-037 - I can see transactions that were rejected by the wallet user (me)
+    test.fail()
```

Comment:
this runs the test and expects it to fail, playwright will complain when it starts passing, which will mean the bug has been fixed and this can be changed

Reply:
`test.fixme('should list...` will just skip this test during test run

Reply:
is that preferable? genuine question. Instead of skipping it outright, if this test starts passing (which it hopefully will when the bug is fixed) it forces you to set it as normal. Looked like the suggested way when I googled but in all honesty I am not hugely fussed either way

Reply:
If `.fixme` is used, then the test is marked as skipped - 
<img width="1033" alt="image" src="https://user-images.githubusercontent.com/24473278/231702816-11045093-7167-4a6d-8e67-576b48f1cee5.png">

I believe there are some better solutions that would allow us to track it better, but that would require some more research and implementation.

Reply:
![image](https://user-images.githubusercontent.com/37066267/231703434-da1f9c78-2c55-4288-9b8b-0c814717493e.png)

So the reporting from playwright shows the fail annotation (fix me does the same, shows fixme), the main difference is that the test actually runs and shows what is currently failing, so when the bug is fixed hopefully that assertion will pass :) hope that helps

Reply:
<img width="557" alt="image" src="https://user-images.githubusercontent.com/24473278/231703659-b07e639d-c507-4868-a57e-24dbb82b7d8a.png">